### PR TITLE
Rename rackspace email mailbox

### PR DESCRIPTION
**Pull request by WilliamLoy**
Please do not merge. Submitting PR to stage article first.

### CODE REVIEW

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/support.rackspace.com/build-e78ea1d12b/how-to/)

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-7d3ecdc988/how-to/)

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-ef49932dc1/how-to/). :bow:

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-579ac36d50/how-to/)

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-c52f4c59a9/how-to/).

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-21bc7c7e0b/how-to/)

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-c7417a067d/how-to/)

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/support.rackspace.com/build-7f5190be81/how-to/)

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-a74b2ce19a/how-to/)

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/support.rackspace.com/build-a32fbd0965/how-to/)

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/support.rackspace.com/build-9664a57e9a/how-to/)

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/support.rackspace.com/build-2a71b1f9bf/how-to/)

**Comment by kmsholcomb** [comment]
Hi @DonNateR. Edits are attached. Let me know if you have any questions. 

[Rename a Rackspace Email address_editsKH.docx](https://github.com/rackerlabs/rackspace-how-to/files/1338345/Rename.a.Rackspace.Email.address_editsKH.docx)
[Rename a Rackspace Email address_editsKH.pdf](https://github.com/rackerlabs/rackspace-how-to/files/1338346/Rename.a.Rackspace.Email.address_editsKH.pdf)

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-e047fe6e57/how-to/).

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-1ca76d93c5/how-to/).

File: content/rackspace-email/rename-a-rackspace-email-mailbox.md [hunk 1]
```diff
@@ -1,39 +1,62 @@
...
+For a video tutorial of the following steps, see [Rackspace Email - Adding/Editing/Renaming/Restoring](https://youtu.be/9eBoiox64UE?t=2m17s).
 
 1. Log in to the [Cloud Office Control Panel](https://cp.rackspace.com/).
 
-2. In the Rackspace Email section, click on **Mailboxes**.
+2. In the **Rackspace Email** section, click **Mailboxes**.
+
+3. In the mailbox list, click the link for the email address that you want to rename.
+
+4. Click the **More** button, expanding the dropdown menu and select **Rename Email Address**.
```
Comment (catlook) [inline content/rackspace-email/rename-a-rackspace-email-mailbox.md]: ... menu, and select ...

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-b5851c90e7/how-to/). :bow:

**Pull request merged**